app/mailers/feedback_mailer.rb

```diff
@@ -51,6 +52,10 @@ def yes_or_no(feedback)
     feedback['done_all_needed'] == true ? 'Yes' : 'No'
   end
 
+  def non_live_environment?
+    %w[staging uat localhost].any? { |env| Rails.configuration.x.application.host.include?(env) } ? '- non-live' : ''
```

Comment:
Conventionally, method names ending with a question mark return true or false

Reply:
Thanks, nice catch.  It was boolean originally, then I realised that notify requires text!